worker/deployer/nested.go

```diff
@@ -159,14 +168,51 @@ func NewNestedContext(config ContextConfig) (*nestedContext, error) {
 	return context, nil
 }
 
-func (c *nestedContext) stopUnit(topic string, data interface{}) {
-	unitName, ok := data.(string)
+func (c *nestedContext) stopUnitRequest(topic string, data interface{}) {
+	units, ok := data.(message.Units)
 	if !ok {
-		c.logger.Errorf("data should be a string")
+		c.logger.Errorf("data should be a Units structure")
+	}
+	for _, unitName := range units.Names {
+		if err := c.stopUnit(unitName); err != nil {
+			c.logger.Errorf("%v", err)
+		}
+	}
+}
+
+func (c *nestedContext) startUnitRequest(topic string, data interface{}) {
+	units, ok := data.(message.Units)
+	if !ok {
+		c.logger.Errorf("data should be a Units structure")
+	}
+	for _, unitName := range units.Names {
+		if err := c.startUnit(unitName); err != nil {
+			c.logger.Errorf("%v", err)
+		}
+	}
+}
+
+func (c *nestedContext) unitStatusRequest(topic string, data interface{}) {
```

Comment:
_ interface{}

to make it clear data is not used

I wonder if we want to allow for status of just specific units

Reply:
sure, and I don't think allowing status just for specific units is useful, this is only what units are there for the machine, there won't me many.